src/app/feature/map/dialogs/manual-dialog/manual-dialog.component.html

```diff
@@ -1,32 +1,141 @@
 <section>
   <h1>{{ mainTitle | translate }}</h1>
-  <p>Описание приложения, что оно делает и вкратце какой функционал содержит</p>
+  <p>{{ mainDescription | translate }}</p>
   <h2 class="subtitle">{{ interfaceTitle | translate }}</h2>
-  <hr />
-  <p>
-    Далее следует описание элементов интерфейса и их функционала по следующему
-    шаблону:
-  </p>
-  <p>- Подзаголовок из названия</p>
-  <p>- Скриншот элемента</p>
-  <p>- Описание местоположения на экране, как он работает и для чего нужен</p>
-  <p>- (Опционально) дополнительные скриншоты меньшего размера</p>
+  <hr/>
+  <div class="interface-container">
+    <ng-container *ngFor="let item of interfaceItems">
+      <div class="interface-item">
+        <p class="interface-item__title">{{ item.tittle | translate }}</p>
+        <img class="interface-item__img" src="{{ item.url }}" alt="">
+        <p class="interface-item__description">{{ item.description | translate }}</p>
+      </div>
+    </ng-container>
+  </div>
+
   <h2 class="subtitle">{{ authorsTitle | translate }}</h2>
-  <hr />
-  <ul>
-    <li>Бида Богдан</li>
-    <li>Котелевец Егор</li>
-    <li>Поляков Денис</li>
-    <li>Тягло Евгений</li>
-  </ul>
+  <hr/>
+  <div class="authors">
+    <div class="author">
+      <div class="author__circle">
+        <svg x="0px" y="0px" viewBox="0 0 101 101" xml:space="preserve">
```

Comment:
Please use components for it
We don`t use img tags because it will be cause of excessive requests, i`m planning to create one component with ngSwitch in feature